DropManager/tests/DropManager-specs.js

```diff
@@ -97,11 +113,175 @@ describe('DropManager Specs', () => {
 					</ResponsiveLayout>
 				</center>
 				<bottom />
-			</Component>
+			</ResponsiveComponent>
 		);
 
 		const responsiveLayout = screen.getByTestId('center').parentElement.parentElement;
 
 		expect(responsiveLayout).toHaveStyle({alignItems: 'center', justifyContent: 'space-evenly'});
 	});
+
+	test('should rearrange items on touch', () => {
+		const arrangement = {bottom: 'bottom', center: 'center', top: 'top'};
+
+		render(
+			<Component arrangeable arrangement={arrangement} data-testid="dropManager">
+				<top is="custom">
+					<div>Drag me top</div>
+				</top>
+				<center is="custom">
+					<div>Drag me center</div>
+				</center>
+				<bottom is="custom">
+					<div>Drag me bottom</div>
+				</bottom>
+			</Component>
+		);
+
+		const topSlot = screen.getByTestId('dropManager').children[0];
+		const centerSlot = screen.getByTestId('dropManager').children[1];
+		const bottomSlot = screen.getByTestId('dropManager').children[2];
+
+		topSlot.getBoundingClientRect = jest.fn(() => {
+			return {
+				width: 501,
+				height: 100,
+				top: 0,
+				left: 0,
+				bottom: 0,
+				right: 0
+			};
+		});
+
+		centerSlot.getBoundingClientRect = jest.fn(() => {
+			return {
+				width: 501,
+				height: 100,
+				top: 101,
+				left: 0,
+				bottom: 0,
+				right: 0
+			};
+		});
+
+		bottomSlot.getBoundingClientRect = jest.fn(() => {
+			return {
+				width: 501,
+				height: 100,
+				top: 202,
+				left: 0,
+				bottom: 0,
+				right: 0
+			};
+		});
+
+		const delta = {x: 0, y: 160};
+		const from = getElementClientCenter(topSlot);
+		const to = {x: from.x + delta.x, y: from.y + delta.y};
+		const step = {x: (to.x - from.x), y: (to.y - from.y)};
+		const current = {clientX: from.x, clientY: from.y};
+
+		fireEvent.touchStart(topSlot, {touches: [current]});
+
+		document.elementFromPoint = jest.fn(() => {
```

Comment:
function had to be manually declared because jsdom does not support it by default